util/chplenv/chpl_home_utils.py

```diff
@@ -17,6 +17,11 @@ def get_chpl_home():
         chpl_home = dirname(dirname(dirname(os.path.realpath(__file__))))
     return chpl_home
 
+@memoize
+def get_chpl_third_party():
```

Comment:
We might one day but at present I don't think any of these Python chplenv scripts would use it and the only reason it's here is to enable that.